### PR TITLE
Added live pipeline logging

### PR DESCRIPTION
**Pull request by LinfanS**
Hi,

This small pull request is for the addition of logging statements (and removal of print statements) in the `live_pipeline.py` file, easiest way to configure this for now was to initialise loggers in both pipeline files with a logger of the same name that is referenced in other files (e.g. `transform.py`) that uses logging.

### CODE REVIEW

File: pipeline/live_pipeline.py [hunk 1]
```diff
@@ -85,23 +82,37 @@ def upload_log_to_s3(aws_client: client) -> None:
...
     live_url = construct_live_url(log_date)
 
     max_page_num = get_max_page_num(live_url)
     if max_page_num == 0:
-        return "No new data to insert, exiting"
+        logger.info("No new data to insert, exiting")
+        return None
 
-    process_pages(live_url, max_page_num, log)
+    count = 0
```
Comment (MilesDrabwell) [inline pipeline/live_pipeline.py:96]: Maybe a more specific count variable name?

File: pipeline/live_pipeline.py [hunk 2]
```diff
@@ -85,23 +82,37 @@ def upload_log_to_s3(aws_client: client) -> None:
...
+    for page_num in range(1, max_page_num + 1):
+        gpt_response = []
+        data = get_listing_data(live_url, page_num, log)
+        count += len(data)
+        for index, _ in enumerate(data):
```
Comment (MilesDrabwell) [inline pipeline/live_pipeline.py:101]: What is the "_" referencing here?

**Review by MilesDrabwell — APPROVED** [review]
Effective logging.

File: pipeline/live_pipeline.py [hunk 3]
```diff
@@ -85,23 +82,37 @@ def upload_log_to_s3(aws_client: client) -> None:
...
+    logger = initialise_logger()
     aws_client = get_client()
     log_json = read_from_json(aws_client)
     log_date, log = extract_log_and_date(log_json)
     live_url = construct_live_url(log_date)
 
     max_page_num = get_max_page_num(live_url)
     if max_page_num == 0:
-        return "No new data to insert, exiting"
+        logger.info("No new data to insert, exiting")
```
Comment (alabbahs) [inline pipeline/live_pipeline.py:93]: nice change from returning to logging instead, and then returning None

**Review by alabbahs — APPROVED** [review]
Again, clean switch to logging from print statements.

File: pipeline/live_pipeline.py (hunk 1 above)
Comment (LinfanS) [reply pipeline/live_pipeline.py:96]: Done, thanks Miles

File: pipeline/live_pipeline.py (hunk 2 above)
Comment (LinfanS) [reply pipeline/live_pipeline.py:101]: It's an empty placeholder which is typical convention for when you don't need a variable. Of course I could've just use range(len) but enumerate was kept to retain consistency with batch_pipeline.

**Pull request merged**